core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -483,7 +483,13 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_DEFAULT_SEQ_PART_TXN_COUNT("cairo.default.sequencer.part.txn.count"),
     POSTHOG_API_KEY("posthog.api.key"),
     POSTHOG_ENABLED("posthog.enabled"),
-    LOG_SQL_QUERY_PROGRESS_EXE("log.sql.query.progress.exe");
+    LOG_SQL_QUERY_PROGRESS_EXE("log.sql.query.progress.exe"),
+    CAIRO_PARTITION_ENCODER_VERSION("cairo.partition.encoder.version"),
+    CAIRO_PARTITION_ENCODER_STATISTICS_ENABLED("cairo.partition.encoder.statistics.enabled"),
```

Comment:
nit: I'd expect to see PARQUET somewhere in the name

Reply:
done